mkdocs.yaml

```diff
@@ -89,6 +90,12 @@ markdown_extensions:
         - name: mermaid
           class: mermaid
           format: !!python/name:pymdownx.superfences.fence_code_format
+  - pymdownx.tabbed:
+      alternate_style: true
+  - pymdownx.highlight:
+      linenums: true
+  - pymdownx.inlinehilite
+  - pymdownx.snippets
```

Comment:
@CBroz1 Could you elaborate on what these mkdocs features do for those not familiar? Perhaps we can add it to the PR description or as inline comments.

Reply:
Certainly - Just added inline comments, and commented out items I not actually used in this PR